compiler/rustc_borrowck/src/diagnostics/move_errors.rs

```diff
@@ -401,7 +401,7 @@ impl<'a, 'tcx> MirBorrowckCtxt<'a, 'tcx> {
         };
         if let Some(use_spans) = use_spans {
             self.explain_captures(
-                &mut err, span, span, use_spans, move_place, "", "", "", false, true,
+                &mut err, span, span, use_spans, move_place, false, false, false, false, true,
```

Comment:
I wonder if we should modify `explain_captures` to take some enums with `Yes`/`No` variants rather than all these boolean arguments.

Reply:
I just made a struct..